TurkishFilter/sections/specific.txt

```diff
@@ -42,6 +42,7 @@ yemektarifleri.org,lifebursa.com,mshowto.org,macizlee1.com,macizlee.ne,dunyaatla
 fullhdfilmcibaba.com,720pizlemek.com,1080pfilmizle.net,fullhdfilmcidayi.com,hdfilmizledayi.com,720-pizleme.com,fullhdfilmizlebaba.com,evrenselfilmlerim.net###adStarter
 !
 forum.donanimhaber.com##div[id^="dha_"]
+hrdergi.com##a[href^="https://globalhrsummit.org"]
```

Comment:
it seems `a[data-ad-id]` is better

Reply:
It doesn't work.

Reply:
weird. I hope you checked it with `##`?:) `hrdergi.com##a[data-ad-id]`
![image](https://user-images.githubusercontent.com/8361299/111306187-95044700-8660-11eb-8178-16d38d2de8c8.png)

Reply:
Yeah, it works somehow now.